pkgs/development/r-modules/default.nix

```
@@ -1123,6 +1123,15 @@ let
         '';
     });
 
+    findpython = old.findpython.overrideAttrs (attrs: {
+      postPatch = ''
+        substituteInPlace "R/find_python_cmd.r" \
+          --replace-fail "python_cmds[which(python_cmds != \"\")]" \
+          "python_cmds <- c(python_cmds, \"${lib.getBin pkgs.python3}/bin/python3\")
```

```suggestion
          "python_cmds <- c(file.path(\"${lib.getBin pkgs.python3}\", \"bin\", \"python3\"), python_cmds)
```

- Changes the search order: if nix python is sufficient, use that. Otherwise, feel free to go wild
- using file.path: the `python_cmds` are already normalized, so it doesn't rely on `/` as a path separator

---

I think the nixpkgs python should be a fallback, if there's another python available it's presumably there intentionally.

---

done